_pages/user-experience/knowledge-articles.md

```diff
@@ -15,9 +15,54 @@ sidenav:
     href: "user-experience/failure-proof/"
   - text: Knowledge articles
```

Comment:
Should this menu option be changed to Customer support? Knowledge articles is only one aspect of the information.

Reply:
i have no opinion about this! @nprimak ?